.github/workflows/tests.yml

```
@@ -109,7 +109,7 @@ jobs:
           persist-credentials: false
 
       - name: Set up Just
-        uses: extractions/setup-just@95b912dc5d3ed106a72907f2f9b91e76d60bdb76  # 1.4
+        uses: extractions/setup-just@69d82fb0233557aec017ef13706851d0694e0f1d  # 1.4
```

```suggestion
      - uses: extractions/setup-just@69d82fb0233557aec017ef13706851d0694e0f1d
```
version number does not match hash